Diagnostic/watcherutil.py

```diff
@@ -19,8 +19,9 @@
 class Watcher():
     lastModTime = os.path.getmtime('/etc/fstab')
```

Comment:
This caught my eyes just now (sorry for being slow), but I wonder if you wanted this to be an instance variable, not a class variable...? As far as I understand, declaring a variable inside a class, but outside __init__() will make it a class variable, probably initialized only once at the script loading time. Is that what you wanted? Of course I do see that in handle_fstab() below, you do assign something to self.ladModTime, so it might work, but I'm not sure if this variable decl/def is right to be here. Same with the "logger" class variable below...